src/components/atoms/text-input/text-input.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import StyledInput from './text-input.styles';
+
+const TextInput = React.forwardRef(
```

Comment:
Like `Button`, I ran into issues and deferred this. There's an issue with how I was handling the `ref` that wasn't playing nice with react-hook-form and I wanted to get the prototype working before I committed to debugging.